docker-compose-base.yaml

```diff
@@ -0,0 +1,53 @@
+services:
+  media-nginx:
+    restart: unless-stopped
+    build:
+      context: nginx
+      dockerfile: nginx-dockerfile
+  media:
+    restart: unless-stopped
+    build:
+      context: .
+      dockerfile: base-dockerfile
+    healthcheck:
+      test: wget http://localhost:8080/graphiql || exit 1
```

Comment:
didn't you add a real health endpoint?

Reply:
(you probably also want to fix that at all your other services)
also use the correct flags: `test: wget -qO - http://localhost:8080/health || exit 1`
otherwise this creates a file each second, and you will run out of inodes earlier or later (I learned this the hard way with Gropius)

Reply:
Yes that would be problematic, i can fix that and use the now existing health endpoint

Reply:
I also added the `wget` package, as this was missing for the service to do the health check. Will write a Gropius issue for the other services